llarp/service/endpoint.cpp

```diff
@@ -914,7 +915,24 @@ namespace llarp
         RemoveConvoTag(frame.T);
         return true;
       }
-      return frame.AsyncDecryptAndVerify(EndpointLogic(), p, m_Identity, this);
+      if (not frame.AsyncDecryptAndVerify(EndpointLogic(), p, m_Identity, this))
+      {
+        // send reset convo tag message
+        ProtocolFrame f;
+        f.R = 1;
+        f.T = frame.T;
+        f.F = p->intro.pathID;
+
+        f.Sign(m_Identity);
```

Comment:
Is this (and should it be) doing crypto in the logic thread?

Reply:
not frequent enough to warrant it imo

Reply:
ok, just checking